website/src/commands/sync.md

```diff
@@ -57,6 +57,17 @@ developers.
 Use the `--dry-run` flag to test-drive this command. It prints the Git commands
 that would be run but doesn't execute them.
 
+### --strategy &lt;name&gt;
```

Comment:
Please update this heading to match the new style (suggested change below), move this option between `--stack` and `--detached`, and add `[--strategy <name>]` to the command summary in the same position.

```suggestion
#### `--strategy <name>`
```

Reply:
Looks great! 🎉